bluebottle/common/management/commands/makemessages.py

```diff
@@ -14,13 +14,15 @@ class Command(BaseCommand):
     ]
 
     def handle(self, *args, **kwargs):
-        strings = []
-        for app, file in self.fixtures:
-            with open('bluebottle/{}/fixtures/{}'.format(app, file)) as fixture_file:
-                strings += [fixture['fields']['name'].encode('utf-8') for fixture in json.load(fixture_file)]
-
         with tempfile.NamedTemporaryFile(dir='bluebottle', suffix='.py') as temp:
-            temp.write('\n'.join(['gettext("{}")'.format(string) for string in strings]))
+            for app, file in self.fixtures:
+                with open('bluebottle/{}/fixtures/{}'.format(app, file)) as fixture_file:
+                    for string in [
+                            fixture['fields']['name'].encode('utf-8')
+                            for fixture
+                            in json.load(fixture_file)]:
```

Comment:
maybe break this up for readability?
strings = fixture['fields']['name'].encode('utf-8') for fixture in json.load(fixture_file)]
for string in strings:
Or something like that?